README.md

```diff
@@ -68,6 +68,7 @@ TBD.
 |:------|:--------:|:---------:|:------------------:|:----------:|:----------------:|:----------------:|
 | T470p | Noteboook | Yes | No | 12.x | Unknown | No |
 | T470 | Noteboook | Yes | No | 11.8 | Unknown | No |
+| X1 Carbon Gen 4 | Noteboook | Yes | No | Unknown | Unknown | No |
```

Comment:
Commit title and entry don't match. Same with one of the other two

Reply:
Done